### DISCUSSION TITLE
help related hacktoberfest

### DISCUSSION THREAD

**Discussion by devesh-mishra13** [freeCodeCamp/Developer_Quiz_Site#773] [Q&A] [answered]
Can anyone please help me by clearing my confusion as i have successfully did a pull request which was later on merged in this Developer_Quiz_Site will this merge request be considered for hacktoberfest

**Comment by jdwilkin4** [comment, marked as answer]
HI @devesh-mishra13 ! 

It looks like you created your first PR here but it hasn't been reviewed yet.
Once it is reviewed and approved and merged, then it will count for hacktoberfest
https://github.com/freeCodeCamp/Developer_Quiz_Site/pull/770

Any repo that has the hacktoberfest topic is participating in hacktoberfest.
If you have 4 successfully merged PR's in any hacktoberfest repo, then you will receive a special digital badge for completing hacktoberfest

You can learn more about the rules on the official website
https://hacktoberfest.com/participation/

Hope that helps
